svelte: Improve syntax highlighting

soerenmeier opened the pull request:
This PR fixes `<script context="module">` not being highlighted.  
It also adds support for scss.

Release Notes:

- N/A

mtorromeo commented on extensions/svelte/languages/svelte/injections.scm:
```diff
@@ -1,32 +1,71 @@
   (raw_text) @content
   (#set! "language" "css"))
 
+; match script tags without a lang tag
```
```suggestion
; match style tags without a lang tag
```

soerenmeier replied:
Thanks, fixed them

mtorromeo commented on extensions/svelte/languages/svelte/injections.scm:
```diff
@@ -1,32 +1,71 @@
+  (#not-eq? @_name "lang")
+  (#set! "language" "css"))
+
+; match javascript
```
```suggestion
; match css
```

mtorromeo commented on extensions/svelte/languages/svelte/injections.scm:
```diff
@@ -1,32 +1,71 @@
+  (#eq? @_value "css")
+  (#set! "language" "css"))
+
+; match typescript
```
```suggestion
; match scss
```

soerenmeier committed:
svelte: Improve syntax highlighting (#10893)

This PR fixes `<script context="module">` not being highlighted.
It also adds support for scss.

maxdeviant reviewed (approved):
Thanks!
